app/interactors/create_order.rb

```diff
@@ -16,10 +16,6 @@ def rollback
     end
   end
 
-  def rollback_order(order)
-    OrderNumber.relinquish(order.order_number)
-  end
-
```

Comment:
Are you sure dropping "relinquishing" order numbers is okay?

Reply:
Yes, moving it into the transaction takes care of this.